app/assets/stylesheets/components/_swiper.scss

```diff
@@ -0,0 +1,18 @@
+.swiper-button-prev, .swiper-button-next {
```

Comment:
I would give some shadow if possible. In the white picture the icons are not very visible

Reply:
The one of the left is intended to be less opaque because it's the first picture.
Do you feel like the right arrow isn't bright enough that you see it easily while making it discret enough to not disturb you ?

I can probably increase the opacity or add a shadow if that's the case then, sure